janusgraph-core/src/main/java/org/janusgraph/diskstorage/configuration/ConfigElement.java

```diff
@@ -78,6 +78,10 @@ public String toString() {
         return (namespace !=null? namespace.toString()+SEPARATOR:"") + name;
     }
 
+    public String toStringWithoutRoot() {
```

Comment:
Is there a scenario where `toString()` makes sense with root? Confusion around `root.*` came up recently on the [mailing list](https://groups.google.com/d/msg/janusgraph-users/dOpxRbi-ZQI/xWYv4T1MAgAJ). Leave it for now, but this looks like a cleanup item for the future.